ddtrace/contrib/structlog/patch.py

```diff
@@ -0,0 +1,88 @@
+import structlog
+from wrapt import wrap_function_wrapper as _w
+
+import ddtrace
+from ddtrace import config
+
+from ...internal.utils import get_argument_value
+from ...internal.utils import set_argument_value
+from ..trace_utils import unwrap as _u
+
+
+RECORD_ATTR_TRACE_ID = "dd.trace_id"
+RECORD_ATTR_SPAN_ID = "dd.span_id"
+RECORD_ATTR_ENV = "dd.env"
+RECORD_ATTR_VERSION = "dd.version"
+RECORD_ATTR_SERVICE = "dd.service"
+RECORD_ATTR_VALUE_ZERO = "0"
+RECORD_ATTR_VALUE_EMPTY = ""
```

Comment:
These constants are duplicates. We can move them to ddtrace/contrib/logging/constants.py

Reply:
Done